crates/robespierre-fw-macros/src/lib.rs

```diff
@@ -25,7 +25,7 @@ pub fn command(
     let result = quote! {
         #command_func
 
-        fn #old_name <'a> (
+        pub fn #old_name <'a> (
```

Comment:
I think it'd be better if we just used the visibility, and make the `__name_impl` private, as that is an implementation detail and should not be exported either way.

```suggestion
    let visibility = std::mem::replace(&mut command_func.vis, syn::Visibility::Inherited);

    let result = quote! {
        #command_func

        #visibility fn #old_name <'a> (
```